### ISSUE TITLE
Automated checks

### ISSUE THREAD

**Issue by smeragoel** [Quansight-Labs/czi-scientific-python-mgmt#55]
## 👥🪪 duplicate-id-aria:
duplicate-id-aria: IDs used in ARIA and labels must be unique - [WCAG 4.1.1](https://www.w3.org/WAI/WCAG21/Understanding/parsing.html)

- [ ] Instance 1
  - **Path**
    `.me-auto > .navbar-item:nth-child(1) > .version-switcher__container.dropdown > .version-switcher__button[role="button"][aria-haspopup="listbox"]`
  
  - **Snippet**
   ```html
  <button id="versionswitcherbutton" type="button" role="button" class="version-switcher__button btn btn-sm navbar-btn dropdown-toggle" data-bs-toggle="dropdown" aria-haspopup="listbox" aria-controls="versionswitcherlist" aria-label="Version switcher list" data-active-version-name="dev" data-active-version="dev">
  ```

- [ ] Instance 2
  - **Path**
    `.me-auto > .navbar-item:nth-child(1) > .version-switcher__container.dropdown > .version-switcher__menu.list-group-flush.py-0`
  
  - **Snippet**
   ```html
  	<div id="versionswitcherlist" class="version-switcher__menu dropdown-menu list-group-flush py-0" role="listbox" aria-labelledby="versionswitcherbutton">
  ```

- [ ] Instance 3
  - **Path**
    `.me-auto > .navbar-item:nth-child(2) > nav > .bd-navbar-elements.navbar-nav > .dropdown.nav-item > .dropdown-menu`
  
  - **Snippet**
   ```html
  <ul id="pst-header-nav-more-links" class="dropdown-menu">
  ```
## 🔗 link-name
link-name: Links must have discernible text - [WCAG 2.4.4](https://www.w3.org/WAI/WCAG21/Understanding/link-purpose-in-context.html), [WCAG 4.1.2](https://www.w3.org/WAI/WCAG21/Understanding/name-role-value.html)

- [ ] Instance 1
  - **Path**
    `.sd-stretched-link.external[href$="pydata.html"]`
  
  - **Snippet**
   ```html
  <a class="sd-stretched-link reference external" href="examples/pydata.html"></a>
  ```

- [ ] Instance 2
  - **Path**
    `.sd-stretched-link.external[href$="gallery.html"]`
  
  - **Snippet**
   ```html
  	<a class="sd-stretched-link reference external" href="examples/gallery.html"></a>
  ```

**Comment by gabalafou** [comment]
I think there are PRs that address everything in this issue. 

Is there a way to mark this issue as "probably done, let's check once all of my PRs have been merged"?

**Comment by smeragoel** [comment]
Hmm, I don't think we have a label for that 😹 Let's link those PRs here, and we will come back to this when they are merged. Your comment should be mroe than enough to remind us to double check!
